### PR TITLE
chore: embedded the contents of ipay88-my.aar

### PR DESCRIPTION
**Pull request by vasi-twks**
A bit of context about this PR. 
Ipay88 do not host their aar on a repository manager, therefore from what I understood from Semir, we received an .aar from them, and we hosted that aar into artifactory then we loaded that dependency from artifactory. 
As the goal is now to remove artifactory, we had two options. 

- host that aar in maven along with the other dependencies (but the old POM is missing a lot of entries and GPG signing)
- embed the aar into the wrapper (actually embed its contents as we cannot embed an aar into another aar)

The second version was chosen so this PR has inlined the required resources and the classes file. 
From what I can tell, nol-pay is another wrapper that has embedded the dependencies this way. 

From a testing perspective, a version suffixed with "-SNAPSHOT" was published into maven local, then added a dependency for the primer-sdk-android project. 
The project compiles and runs. 
I attempted to do a payment flow, with both the dependency library and the embedded library, the result was failed for both of them but that seems to be related to a server workflow.

### CODE REVIEW

**Review by HarryPrimer — APPROVED** [review]

**Comment by FlaviuExtPrimer** [comment]
Fyi, don't forget about ipay88's manifest. Needs to be merged into the wrapper one (I remember there were some permissions)

File: ipay88/build.gradle [hunk 1]
```diff
@@ -6,8 +6,9 @@ plugins {
 }
 
 android {
-    namespace 'io.primer.ipay88'
-    compileSdkVersion 34
+    // in order not to crash their SDK, we can use their namespace to correctly find R class
+    namespace 'com.androids.ipay'
```
Comment (semirp) [inline ipay88/build.gradle:10]: There are solutions for this:

- unpack their code and load resources dynamically, then package it again
- unpack their code and put it inside our package - but some things should not be pushed or added to public repo

At the end this seems as the easiest solution.

File: ipay88/build.gradle [hunk 2]
```diff
@@ -38,9 +39,15 @@ android {
     }
 }
 
+tasks.configureEach { task ->
+    if (task.name == "generateReleaseBuildConfig") {
```
Comment (semirp) [inline ipay88/build.gradle:43]: We have to disable generation of new `BuildConfig` file as gradle does not allow duplicates.

**Comment by semirp** [comment]
> Fyi, don't forget about ipay88's manifest. Needs to be merged into the wrapper one (I remember there were some permissions)

Yeah, all of the permissions and activities were added to the new Manifest 👍

**Pull request merged**